libpod/container_inspect.go

```diff
@@ -885,3 +868,29 @@ func (c *Container) inHostPidNS() (bool, error) {
 	}
 	return true, nil
 }
+
+func (c *Container) GetDevices(hostConfig define.InspectContainerHostConfig, ctrSpec spec.Spec, deviceNodes map[string]string) ([]define.InspectDevice, error) {
```

Comment:
```suggestion
func (c *Container) GetDevices(privileged bool, ctrSpec *spec.Spec, deviceNodes map[string]string) ([]define.InspectDevice, error) {
```
Please don't pass in a full struct when you only need one field.